tests/test_dom0_config.py

```diff
@@ -1,54 +0,0 @@
-import subprocess
-import unittest
-
-DEPRECATED_TEMPLATES = [
```

Comment:
While this test could be useful to reinstate in future, it currently only tests for deprecations which should no longer be encountered in any install given the 4.1 "refresh". So I think it's safe to remove in its entirety and best to reinstate if/when we do the next template reorganization.

Reply:
looks like the `test_vms_to_update_are_tagged` is still valid right? it does help to clear up the rest of this though.

Reply:
This test specifically checks for the presence of the `sd-workstation-updates` tag, which was used by the legacy cron job updater to enumerate the list of VMs to update, so I think it is safe to remove. Note that we have a separate test to verify that all VMs are updated, which is here: https://github.com/freedomofpress/securedrop-workstation/blob/main/tests/test_vms_platform.py#L84